internal/controller/operatorconfigmap_controller.go

```diff
@@ -410,7 +411,11 @@ func (c *OperatorConfigMapReconciler) reconcileDelegatedCSI() error {
 		if c.AvailableCrds[VolumeGroupSnapshotClassCrdName] {
 			csiOperatorConfig.Spec.DriverSpecDefaults.SnapshotPolicy = csiopv1a1.VolumeGroupSnapshotPolicy
 		}
-
+		csiCtrlPluginHostNetwork, err := strconv.ParseBool(c.operatorConfigMap.Data[csiCtrlPluginHostNetworkKey])
+		if err != nil {
+			return fmt.Errorf("failed to parse %s: %v", csiCtrlPluginHostNetworkKey, err)
+		}
```

Comment:
There is no point in handling parsing errors here. We can just treat them as false
```suggestion
		csiCtrlPluginHostNetwork, _ := strconv.ParseBool(c.operatorConfigMap.Data[csiCtrlPluginHostNetworkKey])

```